### PR TITLE
Add support for 320x200 256 color VGA video mode 

### PR DESCRIPTION
**Pull request by phil-opp**

### CODE REVIEW

File: src/video_mode/vga_320x200.s [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    mov eax, vga_position
+    mov ecx, 80 * 2
+    div ecx
+    add eax, 1
+    mul ecx
+    mov vga_position, eax
+
+    pop edx
+    pop ebx
+    pop ecx
```
Comment (bjorn3) [inline src/video_mode/vga_320x200.s]: you swapped ebx and ecx

File: src/video_mode/vga_text_80x25.s [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    mov eax, vga_position
+    mov ecx, 80 * 2
+    div ecx
+    add eax, 1
+    mul ecx
+    mov vga_position, eax
+
+    pop edx
+    pop ebx
+    pop ecx
```
Comment (bjorn3) [inline src/video_mode/vga_text_80x25.s]: same here

**Comment by phil-opp** [comment]
@bjorn3 Thanks for the review!

bors r+

**Comment by phil-opp** [comment]
Published as version 0.3.7

**Pull request merged**